frontends/verilog/verilog_parser.y

```diff
@@ -2530,6 +2531,7 @@ gen_stmt:
 		ast_stack.push_back(node);
 		ast_stack.back()->children.push_back($3);
 	} gen_stmt_block opt_gen_else {
+		SET_AST_NODE_LOC(ast_stack.back(), @1, @6);
```

Comment:
Again, if I'm counting correctly, `@6` should be `gen_stmt_block`, not `opt_gen_else`.  Is this intentional?  Why is `opt_gen_else` not counted as part of the location for the `AST_GENIF`?  Does that `opt_gen_else` have its own cell type that gets its location properly set?

I'm not saying this is wrong; I honestly don't know.